lib/methwhy.g

```diff
@@ -89,12 +89,89 @@ end);
 ##      also for each not applicable method all reasons why it is not
 ##      applicable.
 ##  </Item>
-##  <Mark>6</Mark>
-##  <Item>
-##      also the function body of the selected method(s)
-##  </Item>
 ##  </List>
 ##  <P/>
+##  Since <C>ApplicableMethod</C> returns a function, <C>Print(last);</C> 
+##  may be used to view the code. 
+##  <P/>
+##  If the first argument is a function, rather than an operation, 
+##  then the function is returned. 
```

Comment:
I wonder what the motivation for this is. Wouldn't it be better to have an error that says that ApplicableMethod may only be used on operations, but not on plain functions?

Reply:
BTW, note that operations are also functions, so I find the current sentence rather confusing.

Reply:
It seems best to avoid ApplicableMethod throwing an error - just return 'fail' as appropriate. 
The occasional user may not realise that the function they are interested in (e.g. DirectProduct) is not an operation, so it is helpful to return that function so that a quick look at the listing by the user shows what operation(s) is called (e.g. DirectProductOp).
"rather than an operation" now changed to "but not an operation".